lbcaccessibility/src/main/java/studio/lunabee/compose/lbcaccessibility/extension/ContextExt.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright © 2022 Lunabee Studio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * ContextExt.kt
+ * Lunabee Compose
+ *
+ * Created by Lunabee Studio / Date - 5/10/2022 - for the Lunabee Compose library.
+ */
+
+package studio.lunabee.compose.lbcaccessibility.extension
+
+import android.content.Context
+import android.content.Intent
+import android.provider.Settings
+
+/**
+ * Open phone settings to accessibility page.
+ */
+fun Context.openAccessibilitySettings() {
+    val intent = Intent(Settings.ACTION_ACCESSIBILITY_SETTINGS)
+    startActivity(intent)
+}
```

Comment:
Not really linked to compose, and I'm not we need an extension for that (I mean, we could create an extension for every `Settigns.ACTION_XXX`...)